packages/agent-toolkit/src/core/tools/platform-api-tools/create-group/create-group-tool.ts

```diff
@@ -0,0 +1,52 @@
+import { z } from 'zod';
+import {
+  PositionRelative,
+  CreateGroupMutation,
+  CreateGroupMutationVariables,
+} from '../../../../monday-graphql/generated/graphql';
+import { createGroup } from './create-group.graphql';
+import { ToolInputType, ToolOutputType, ToolType } from '../../../tool';
+import { BaseMondayApiTool, createMondayApiAnnotations } from '../base-monday-api-tool';
+
+export const createGroupToolSchema = {
```

Comment:
im shocked there is no param for definiging group name :( (I know it dosent exist)